package.json

```diff
@@ -74,25 +74,14 @@
   },
   "resolutions": {
     "**/@types/node": "^14.17.32",
-    "**/@types/react": "^16.14.23",
-    "**/ejs": "^3.1.6",
     "**/front-matter": "^4.0.2",
     "**/glob-parent": "^6.0.0",
     "**/hoist-non-react-statics": "^3.3.2",
-    "**/immer": "^9.0.6",
     "**/kind-of": ">=6.0.3",
-    "**/lodash": "^4.17.21",
     "**/merge": "^2.1.1",
     "**/minimist": "^1.2.5",
     "**/node-jose": "^2.1.0",
-    "**/normalize-url": "^4.5.1",
-    "**/prismjs": "^1.23.0",
-    "**/react-syntax-highlighter": "^15.3.1",
-    "**/react-syntax-highlighter/**/highlight.js": "^10.4.1",
-    "**/ssri": "^6.0.2",
-    "**/tar": "^6.1.11",
     "**/trim": "^0.0.3",
-    "**/trim-newlines": "^3.0.1",
```

Comment:
This was added for https://github.com/advisories/GHSA-7p7h-4mm5-852v?

So it's no longer needed or wasn't needed?

Reply:
It's no longer needed because of the other upstream dependency changes that have been made since. If you look at the lockfile there are no versions before 3.x that are depended on anymore (this wasn't the case when this line was added).